pkg/core/config/tenant.go

```diff
@@ -896,7 +896,7 @@ func (c NexmoConfiguration) IsValid() bool {
 	return c.APIKey != "" && c.APISecret != ""
 }
 
-type HookAppConfiguration struct {
+type HookTimeoutConfiguration struct {
```

Comment:
I think we may store other hook related configuration here, so maybe use another name?

Reply:
Originally we have `HookAppConfiguration` and `HookUserConfiguration`, how about change it as followed?
`HookAppConfiguration` -> `HookTenantConfiguration`
`HookUserConfiguration` -> `HookAppConfiguration`

Any better suggestion?

Reply:
OK. Let's use Carmen's suggestion.